src/Aquifer.Jobs/Program.cs

```diff
@@ -12,12 +14,15 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
 
 var host = new HostBuilder()
     .ConfigureFunctionsWorkerDefaults()
     .ConfigureServices((context, services) =>
     {
         services.AddOptions<ConfigurationOptions>().Bind(context.Configuration);
+        services.AddSingleton(cfg => cfg.GetService<IOptions<ConfigurationOptions>>()!.Value.OpenAi);
```

Comment:
I moved AI translation into its own project and thus I needed to support options for a class that lives in a separate project from the running app.  To do this I settled on a simple method of:
1. Specifying the class in the dependency project.
2. Loading the options as part of the standard `ConfigurationOptions` object.
3. _Also_ wiring up the specific options as a separate Singleton so that they can be accessed by only the dependency project's class (which doesn't know about `ConfigurationOptions`).

I could also have used `IOptions<T>` but chose this strategy because it is harder to bind to `IOptions<T>` here without doing a rebind or a separate bind for that configuration section (by name) and I don't see a lot of value in the `IOptions<T>` wrapper.